pkg/api/norman/customization/clusterregistrationtokens/import.go

```diff
@@ -30,8 +37,13 @@ func ClusterImportHandler(resp http.ResponseWriter, req *http.Request) {
 		authImage = authImages[0]
 	}
 
+	var cluster *v3.Cluster
+	if clusterID != "" {
+		cluster, _ = ch.Clusters.Get(clusterID, metav1.GetOptions{})
```

Comment:
If there is err should we return a http error instead of continuing?

Reply:
the earlier code always generated agent template, so I wanted to keep that behavior as is even if cluster get fails. the only thing we get from cluster obj is agent env vars, so didn't seem enough to fail, what do you think?

Reply:
:+1: